src/FSharp.Formatting.Common/Menu.fs

```diff
@@ -0,0 +1,37 @@
+module FSharp.Formatting.Common.Menu
+open System.IO
+open FSharp.Formatting.Templating
+
+type MenuItem =
+    {
+        Link: string
+        Content: string
+    }
+//am I suppose to have this global lets?
+let fsdocsMenuHeaderContentKey = ParamKey "fsdocs-menu-header-content"
```

Comment:
You probably want to move these to https://github.com/fsprojects/FSharp.Formatting/blob/139754a685d26badb19c38998f88cc8e1de7dc87/src/FSharp.Formatting.Common/Templating.fs#L29